src/components/photo.tsx

```diff
@@ -1,85 +1,105 @@
 import React from 'react'
 import type { FC } from 'react'
-import { Card, Image } from 'react-bootstrap'
+import { Card, Image, Row, Col } from 'react-bootstrap'
 
 import DateTimeStr from './date_time_str'
 import GpsCoordStr from './gps_coord_str'
 import type PhotoMetadata from '../types/photo_metadata.type'
 
 interface PhotoProps {
     description: React.ReactNode
-    id: string
     label: string
-    metadata: PhotoMetadata
-    photo: Blob | undefined
+    photos: { id: string; photo: Blob; metadata: PhotoMetadata }[] // Array of photo objects with metadata
```

Comment:
What happens to preexisting projects from before this change, where the `PhotoProps.photo` attribute is defined?

Reply:
Thanks for pointing this out. I will work on the fix for this scenario.

Reply:
I have tested this feature with the photos stored before this change. Existing photo will be populated as one photo with the option to delete and 'Add Photo' button.  
PhotoProps.photo is now removed and added as PhotoProps.photos to hold multiple photos and the respective metadata. It is updated accordingly in the wrapper class - https://github.com/pnnl/Quality-Install-Tool/blob/17-improve-photoinput-component/src/components/photo_wrapper.tsx#L132